### PR TITLE
Add tool icons for gatsby, react, cypress and cucumber.

### PR DESCRIPTION
**Pull request by JBartscher**
Add new icon for gatsby, react, cypress and cucumber and complement the new tool icons in the readme.md.

### CODE REVIEW

File: tools/react.puml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+@startuml
+sprite $react [64x64/16] {
+EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE
```
Comment (ppxl) [inline tools/react.puml]: This file does not look right. Did you use an input picture with alpha-channels? The background is supposed to be white/`00` but totally off, almost black (even if it's `EE`)
grafik

File: tools/gatsby.puml [hunk 2]
```diff
@@ -0,0 +1,76 @@
+@startuml
+sprite $gatsby [64x64/16] {
+FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
```
Comment (ppxl) [inline tools/gatsby.puml]: This file does not look right. Did you use an input picture with alpha-channels? The background is supposed to be white/`00` but is actually black/`FF`
grafik

File: tools/cypress.puml [hunk 3]
```diff
@@ -0,0 +1,75 @@
+@startuml
+sprite $cypress [64x64/16] {
+0000000000000000000000000000000000000000000000000000000000000000
```
Comment (ppxl) [inline tools/cypress.puml]: This one looks good. Only the border looks weird but I think no one is going to argue.

File: tools/cucumber.puml [hunk 4]
```diff
@@ -0,0 +1,64 @@
+@startuml
+sprite $cucumber [64x52/16] {
+0000000000000000000000000000000000000000000000000000000000000000
```
Comment (ppxl) [inline tools/cucumber.puml]: The low contrast between the speech bubble gray and the text ".js" makes it bad to read/recognize. Should we enhance the contrast a little bit?

Also we should see to it that all images have the same dimensions, that is 64x64 px.

grafik

**Review by ppxl — CHANGES_REQUESTED** [review]
Thanks for sharing your work! I think we need to correct some image background colors and one image dimension. See the comments.

File: tools/cucumber.puml (hunk 4 above)
Comment (JBartscher) [reply tools/cucumber.puml]: I can try to tweak the contrast. About the dimension of the sprites: In the readme it says 64x64px but many of the other sprites are in 48x48px what is correct here?

File: tools/gatsby.puml (hunk 2 above)
Comment (JBartscher) [reply tools/gatsby.puml]: Yeah I used a .png instead of jpgs. I fix this right away

File: tools/react.puml (hunk 1 above)
Comment (JBartscher) [reply tools/react.puml]: Same here

**Pull request merged**